spec/dummy/Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'rails', '3.2.22.5'
+gem 'rake', '12.2'
```

Comment:
@etagwerker good idea, thanks! We should also make this `"~> 12.2"` instead, so that it fits the development dependency. Otherwise, `rails_upgrader`'s `bundle install` will install the latest `12.3.x` and above release and this will keep using `12.2`.